appveyor.yml

```diff
@@ -37,7 +37,7 @@ build_script:
   - Rscript -e "install.packages('https://github.com/Open-Systems-Pharmacology/OSPSuite-R/releases/download/v10.0.72/ospsuite_10.0.72.zip', repos = NULL, type = 'binary')"
   - Rscript -e "install.packages('https://ci.appveyor.com/api/buildjobs/lq7s2r98966wqpjh/artifacts/tlf_1.2.33.zip', repos = NULL, type = 'binary')"

```

Comment:
@Yuri05 I am installing the packages as zip here. THat;s why it works?

Reply:
Not sure I understand the question...

Reply:
THE PAT File is only used when installing from github which I have removed a couple of hours ago
So the PAT is defacto not used here (as you can see, tlf and ospsuite-r are installed from packages and not from github. So the PAT changes are useless
To test if this works, you need to undo my changes from previous commit in develop
Also personal PAT expires after 30 days? This is not sustainable